source/views/sis/course-search/detail/index.js

```diff
@@ -139,6 +139,24 @@ function Schedule({course}: {course: CourseType}) {
 	)
 }
 
+function Notes({course}: {course: CourseType}) {
+	let notesText = course.notes ? course.notes.join(' ') : ''
+	const notes =
+		Platform.OS === 'ios' ? (
+			<SelectableCell text={notesText} />
+		) : (
+			<Cell
+				cellContentView={
+					<Text selectable={true} style={styles.chunk}>
+						{notesText}
+					</Text>
+				}
+			/>
+		)
+
+	return course.notes ? <Section header="NOTES">{notes}</Section> : null
```

Comment:
At the top: "if not notesText: return null" instead of creating the empty objects.